app/src/main/res/layout/intro_screen_two.xml

```diff
@@ -1,50 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent" android:layout_height="match_parent"
-    android:background="@color/colorPrimary">
-
-
-    <ImageView
-        android:id="@+id/sigin_image_logo"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="80dp"
-        android:layout_centerHorizontal="true"
-        android:src="@mipmap/signin_logo" />
-
-    <TextView
-        android:id="@+id/signin_text_almost_there"
-        android:layout_marginTop="16dp"
-        android:layout_below="@id/sigin_image_logo"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_centerHorizontal="true"
-        android:fontFamily="@font/roboto_medium"
-        android:text="@string/sigin_text_almost_there"
-        android:textColor="#90ffffff" />
-
-    <com.google.android.gms.common.SignInButton
-        android:id="@+id/google_signin_btn"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginEnd="32sp"

```

Comment:
Please remove hardcoded strings from the file and take it from dimens.xml file

Reply:
red colour means deleted. you are commenting on a deleted file.

Reply:
Sorry for that